packages/wpcom-proxy-request/src/index.ts

```diff
@@ -91,48 +112,51 @@ const supportsProgress = !! window.ProgressEvent && !! window.FormData;
 
 debug( 'using "origin": %o', origin );
 
+interface Callback {
+	( ...args: unknown[] ): unknown;
+}
+
 /**
  * Performs a "proxied REST API request". This happens by calling
  * `iframe.postMessage()` on the proxy iframe instance, which from there
  * takes care of WordPress.com user authentication (via the currently
  * logged-in user's cookies).
  *
- * @param {object} originalParams - request parameters
- * @param {Function} [fn] - callback response
- * @returns {window.XMLHttpRequest} XMLHttpRequest instance
+ * @param originalParams - request parameters
+ * @param fn - callback response
```

Comment:
Is `fn` optional or was that a mistake in the original JSDoc?

Reply:
It was, but apparently didn't need to be. This is an internal/private function that always receives a `fn` callback.